src/routes/Settings.tsx

```diff
@@ -230,10 +230,14 @@ export const SettingsRoute: React.FC = () => {
 
           <button
             className={footerButtonClass}
-            title="Logout"
+            title={`Logout ${accounts.user.login}`}
```

Comment:
How about "Logout of "?

Reply:
Logout from?

Reply:
Pretty even - https://trends.google.com/trends/explore?q=Logout%20of,Logout%20from&hl=en

Reply:
https://github.com/gitify-app/gitify/pull/983